README.md

```diff
@@ -42,7 +42,69 @@ PM > pip3 install --upgrade --pre arcus-azureml
+- Following any (optional) extra instructions in the WSL development [environment documentation](https://docs.microsoft.com/en-us/windows/wsl/setup/environment#set-up-your-linux-username-and-password).
+
+### Install and verify required Python prerequisites
+
+- Verify python is installed in your Linux distribution by running `python3 --version`.  This guide has been tested with **Python 3.8.10**.
+- Install **pip**, which is the Python Package installer.  Execute `sudo apt install python3-pip`.
+- Install **venv**, which supports to create Virtual Python Environments (and isolate dependencies from each other).  Execute `sudo apt install python3-venv`.
+- When running `deactivate`, you deactive the virtual environment.
+
+### Create your Arcus python environment
+
+- Get the correct code from Github, by performing a `git clone` into your WSL.
+- Install all required dependencies for your arcus project, by executing the following in your WSL prompt: `pip3 install -r requirements.txt`.  This takes all packages that are configured in the requirements.txt file and installs them in your virtual environment.
+- Execute the following in the WSL shell: `python3 -m venv .venv`.  This creates a new virtual Python environment, in a directory named **.venv**.  This can be done for every Python project you develop.  
+- Activate and start your .arcus environment by running `source .venv/bin/activate`. 
+- Exclude `.venv` from your `.gitignore`, if not done yet. 
+
+### Configure Visual Studio Code
+
+In order to get started developing locally, we recommend [Visual Studio Code](https://code.visualstudio.com/) as the main tool for development.  If you are new to this, we recommend reading this [overview and getting started guide](https://code.visualstudio.com/docs/python/python-tutorial).
+
+The following VSCode extensions are recommended:
+
+- **Remote - WSL Extension**: use Windows Subsystem for Linux as your integrated development environment. ([Install](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl))
+- **Python (Microsoft)**: Intellisense, Test Explorer, Linting & Debugging. ([Install](https://marketplace.visualstudio.com/items?itemName=ms-python.python))
+
+### Open Arcus in Visual Studio Code
+
+- Execute the `Remote-WSL: Open Folder in WSL...` command in VSCode. 
+- Navigate to `/home/<user>` and your correct arcus folder in the WSL folder selection.
+
+The following general Python packages are recommended to install (these are not needed in the requirements.txt file, as they are for the development environment only).
+
+That can be done by executing the following lines in your WSL terminal:
+
+```bash
+pip3 install <package-name>
+```
+
+- **pytest**: for the unit tests
+- **jupyter-notebook**: for the jupyter notebooks to run
+
+### Run unit tests
+
+The VSCode Test Explorer should normally discover and show all unit tests that are part of the solution.  You can easily run those tests as a validation.
```

Comment:
Some visualization is maybe here in order. Some troubleshooting, too, maybe.
What if the tests aren't discovered, what if the build fails, ...
If we force people to use vscode, we should also make sure that they are guided, I think.

Reply:
Again : not forcing people
Will add Test Explorer visualization.  And describe that the output window typically shows all errors, used for troubleshooting